docs/src/content/docs/reference/configuration.md

```diff
@@ -317,3 +317,12 @@ interface HeadConfig {
   content?: string;
 }
 ```
+
+### `showUpdateDate`
```

Comment:
Small API detail: I think it would be nice if the config file and frontmatter options had the same name — I’d suggest `lastUpdated` for both.

It’s been bugging me that I ended up with `editLink` in config and `editUrl` in frontmatter — I might still make a breaking change at some point to align those 😅 

```suggestion
### `lastUpdated`
```